sources/ghostdog/fuzz/fuzz_targets/img_scanning.rs

```diff
@@ -0,0 +1,12 @@
+#![no_main]
+use libfuzzer_sys::fuzz_target;
+
+fuzz_target!(|data: &[u8]| {
+    // Prevent known panic in gptman-1.0.0 with inputs smaller than a sector size
+    if data.len() < 512 {
+        return;
+    }
+
+    let mut data = std::io::Cursor::new(data);
+    let _ = ghostdog::find_device_type(&mut data);
```

Comment:
This doesn't seem to be part of bottlerocket?

Reply:
I have no idea if ghostdog is packaged with the bottlerocket OS, but that doesn't really matter right? It's part of this repo, and it reads external input from files.